File: Leetcode-Solutions/80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int j=0;
        int count=0;
        int last=nums[0];
        for(int i=0;i<nums.length;i++){
        if(nums[i]==last && count<2){
            int temp=nums[i];
        nums[i]=nums[j];
        nums[j]=temp;
            count++;
            j++;
        }
        else if(nums[i]!=last){
            last=nums[i];
            int temp=nums[i];
        nums[i]=nums[j];
        nums[j]=temp;
            j++;
            count=1; 
        }
        }
        return j;
    }
}

File: Leetcode-Solutions/80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        if (nums.Length <= 2)
        {
            return nums.Length;
        }

        int slow = 2;
        for (int fast = 2; fast < nums.Length; fast++)
        {
            if (nums[slow - 2] != nums[fast])
            {
                nums[slow] = nums[fast];
                slow++;
            }
        }

        return slow;
    }
}